test/parallel/test-fs-read.js

```diff
@@ -78,13 +78,20 @@ assert.throws(
   }
 );
 
-['buffer', 'offset', 'length'].forEach((option) =>
+['buffer', 'offset', 'length'].forEach((option) => {
   assert.throws(
     () => fs.read(fd, {
       [option]: null
     }),
     `not throws when options.${option} is null`
-  ));
+  );
+  assert.throws(
+    () => fs.readSync(fd, {
+      [option]: null
+    }),
+    `not throws when options.${option} is null`
+  );
+});
```

Comment:
I think there is a bug in the original test. `fs.read` was throwing for `null` values of `offset` and `length` in the passed `options` object because no callback was passed. The test only works for `options = { buffer: null }`.
Perhaps use this?
```js
assert.throws(
  () => fs.read(fd, {
    buffer: null
  }, () => {}),
  /TypeError: Cannot read property 'byteLength' of null/,
  'not throws when options.buffer is null'
);

assert.throws(
  () => fs.readSync(fd, {
    buffer: null
  }),
  {
    name: 'TypeError',
    message: 'The "buffer" argument must be an instance of Buffer, ' +
    'TypedArray, or DataView. Received an instance of Object',
  },
  'not throws when options.buffer is null'
);
```

Reply:
Oh, it's a good catch. Updated